agent/config/config.go

```diff
@@ -431,6 +469,41 @@ func (a *Agent) Validate() error {
 	return result.ErrorOrNil()
 }
 
+func (d *DynamicApplicationSizing) merge(b *DynamicApplicationSizing) *DynamicApplicationSizing {
+	if d == nil {
+		return b
+	}
```

Comment:
👍 😉

Reply:
It's almost like my previous PRs were a giant ball of mess that I had to split up 😬